changelog.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * Add tags list to filter project.
   * Add search bar for projects.
   * Add search bar for templates.
+* Multi diagrams page:
```

Comment:
Not on this line but, you have to delete the commit with the update dependencies